astro/src/components/Uielement.astro

```diff
@@ -1,6 +0,0 @@
----
```

Comment:
Looks like `Uielement` is still used in quickstart-springboot-web

Reply:
Thanks fixed.